kolibri/plugins/learn/assets/src/views/classes/AssignedExamsTable.vue

```diff
@@ -112,6 +112,16 @@
       },
       examViewerLink,
     },
+    created() {
+      this.resetExamAttempLogs();
+    },
+    vuex: {
+      actions: {
+        resetExamAttempLogs(store) {
+          store.state.examAttemptLogs = [];
```

Comment:
I think this might work in special cases, but let's pretend we're running vuex in "safe" mode, so we can only make changes to `store.state` by dispatching a mutation.